packages/side-nav/src/SideNavGroup.spec.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { render, cleanup, screen, fireEvent } from '@testing-library/react';
+import { SideNavGroup, SideNavItem } from '.';
+import { SideNavGroupProps } from './SideNavGroup';
+
+describe('packages/side-nav', () => {
```

Comment:
can we add a test for when the group label has a glyph, that it's rendered as aria-hidden?

Reply:
I can test that the role is appropriate, but testing aria-hidden is creating a duplicate test for the icon component, not this component.

Reply:
yes wfm